docs/ref/models/meta.txt

```diff
@@ -151,8 +151,9 @@ Retrieving fields composing the primary key of a model
             <django.db.models.fields.IntegerField: id>
         ]
 
-    Otherwise it will contain the single field declared as the
-    :attr:`primary key <django.db.models.Field.primary_key>` of the model.
+    Otherwise it will contain the single field declared as the primary key
+    (:attr:`primary_key=True <django.db.models.Field.primary_key>`) of the
```

Comment:
I think in this case it's less clear as there might not be anything with `primary_key=True` declared on the model